src/cbc_sdk/platform/policies.py

```diff
@@ -582,6 +655,94 @@ def object_rules(self):
             self._object_rules_need_load = False
         return self._object_rules
 
+    @property
+    def object_rule_configs(self):
+        """
+        Returns a dictionary of rule configuration IDs and objects for this Policy.
+
+        Returns:
+            dict: A dictionary with rule configuration IDs as keys and PolicyRuleConfig objects as values.
+        """
+        if self._object_rule_configs_need_load:
+            cfgs = self._info.get("rule_configs", self._info.get("rapid_configs", []))
+            ruleconfigobjects = [PolicyRuleConfig._create_rule_config(self._cb, self, cfg) for cfg in cfgs]
```

Comment:
It's not creating rule configs themselves, it's creating the _objects_ that represent the rule config data _already loaded_ as part of the policy. `_create_rule_config` is a helper function that will ultimately handle creating specialized objects for certain types of rule configs, such as host-based firewall.

Reply:
I couldn't remember if creating a MutableBaseModel also pushed to the backend during the constructor call. It appears that is not the case though I am still hesitant about using the protected method instead of calling the constructor directly

Reply:
Would you prefer the function be called something else, like `_construct_rule_config` maybe?

Reply:
It doesn't need renaming, it needs moving.

Reply:
It got moved.